model_configs/dagrnn_regression_config.json

```diff
@@ -1,5 +1,6 @@
 {
     "__init__": {
+        "basic": false,
```

Comment:
The BasicDAGRNN and DAGRNN are different models. Perhaps they should not be controlled with a single bool in a config file. Rather, let's make them both first class citizens and add the basic one to the test.sh script, give it its own model class, etc.